spot-client/src/common/utils/window-handler.js

```diff
@@ -10,7 +10,22 @@ export default {
      * @returns {string}
      */
     getBaseUrl() {
-        return window.location.origin;
+        const origin = window.location.origin;
```

Comment:
is this code changing on what the function returns ? are the JSDocs still valid here ?

Reply:
Sort of. I actually think they were never really accurate to begin with. I'll tweak.